int16 motion_type
geometry_msgs/Twist cmd_vel
geometry_msgs/Point lookahead_point
geometry_msgs/Point lookahead_point_stairs
geometry_msgs/Point goal_point
